source/__tests__/integration/create-resource.test.js

```diff
@@ -87,3 +157,5 @@ async function expect_value_in_sel_textContent(sel, value) {
   const sel_text = await page.$eval(sel, e => e.textContent)
   expect(sel_text).toBe(value)
 }
+
+
```

Comment:
clearly not a big deal, but ... it's not a great habit to add extra whitespace to no effect ...